harmoni_actuators/harmoni_tts/src/harmoni_tts/local_tts_client.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python3
+
+import os
+import torch
+import time
+
+from TTS.vocoder.utils.generic_utils import setup_generator
+from TTS.utils.generic_utils import setup_model
+from TTS.utils.io import load_config
+from TTS.utils.text.symbols import symbols, phonemes
+from TTS.utils.audio import AudioProcessor
+from TTS.utils.synthesis import synthesis
+
+import sounddevice as sd
+
+
+class TtsClient:
+
+    def __init__(
+        self,
+        tts_config,
+        tts_model,
+        vocoder_config,
+        vocoder_model,
+        use_cuda=False,
+        verbose=False,
+        speedup=1.1
+    ):
+        self._use_cuda = use_cuda
+        self._verbose = verbose
+        self._speedup = speedup
+
+        self._check_files_exist([
+            tts_config, tts_model, vocoder_config, vocoder_model
+        ])
```

Comment:
The config folder appeared to be empty for me.

Reply:
The same. It doesn't exist the file config.json in the model path.

Reply:
@micolspitale93, did you first run `sh harmoni_actuators/harmoni_tts/setup_tts.sh` from the HARMONI directory? This script will download the models and config files.

Reply:
My bad, I'll try it.